mmengine/optim/optimizer/optimizer_wrapper.py

```diff
@@ -305,9 +346,11 @@ def _clip_grad(self) -> None:
         params = list(
             filter(lambda p: p.requires_grad and p.grad is not None, params))
         if len(params) > 0:
-            grad_norm = clip_grad.clip_grad_norm_(params,
-                                                  **self.clip_grad_kwargs)
-            self.message_hub.update_scalar('train/grad_norm', float(grad_norm))
+            grad = self.clip_func(params, **self.clip_grad_kwargs)
+            # `torch.nn.utils.clip_grad_value_` will return None.
+            if grad is not None:
+                self.message_hub.update_scalar(f'train/{self.grad_name}',
```

Comment:
If grad is not None, the `grad_name` must be 'grad_norm' because `torch.nn.utils.clip_grad_value_` will return None. Therefore, there is no need to define `self.grad_name` varibale.